apps/leo_gateway/src/leo_gateway_s3_api.erl

```diff
@@ -412,6 +412,32 @@ head_bucket(Req, Key, #req_params{access_key_id = AccessKeyId}) ->
              {ok, Req} when Req::cowboy_req:req(),
                             Key::binary(),
                             ReqParams::#req_params{}).
+
+%% @doc GET-ACL Operation on Objects
+%% As we only support bucket level ACL, reply bucket ACL here
+%% Related Issue: leo-project/leofs#490
+get_object(Req, Key, #req_params{is_acl = true,
+                                 bucket_name = Bucket}) ->
+    BeginTime = leo_date:clock(),
+    case leo_gateway_rpc_handler:head(Key) of
+        {ok, #?METADATA{del = 0}} ->
+            case leo_s3_bucket:find_bucket_by_name(Bucket) of
```

Comment:
leo_s3_bucket:find_bucket_by_name should implement handling obsolete bucket information as 
https://github.com/leo-project/leofs/issues/642 does in order to keep working while both managers are down.
I'll file this as another issue.